src/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.product__card {
+  display: flex;
+  flex-direction: column;
+  height: 408px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
```

Comment:
add box-sizing to all elements on the page

Reply:
if you add box-sizing with * selector, it will apply to all elements on the page, no need to add it separately